sormas-ui/src/test/java/de/symeda/sormas/ui/importer/CountryImporterTest.java

```diff
@@ -30,29 +30,34 @@
 public class CountryImporterTest extends AbstractBeanTest {
 
 	@Test
-	public void testUmlautsInCountryImport() throws IOException, InvalidColumnException, InterruptedException, CsvValidationException, URISyntaxException {
+	public void testUmlautsInCountryImport()
+		throws IOException, InvalidColumnException, InterruptedException, CsvValidationException, URISyntaxException {
 		TestDataCreator.RDCF rdcf = new TestDataCreator().createRDCF("Default Region", "Default District", "Default Community", "Default Facility");
 		UserDto user = creator.createUser(rdcf.region.getUuid(), rdcf.district.getUuid(), rdcf.facility.getUuid(), "Default", "User", UserRole.ADMIN);
 
 		File countryCsvFile = new File(getClass().getClassLoader().getResource("sormas_country_import_test.csv").toURI());
 		InfrastructureImporter importer = new CountryImporterExtension(countryCsvFile, user);
 		importer.runImport();
-		getCountryFacade().getByDefaultName("Country with ä", false).get(0);
+		CountryReferenceDto countryReference = getCountryFacade().getByDefaultName("Country with ä", false).get(0);
+		assertEquals("XYZ", countryReference.getIsoCode());
 	}
 
-
-	@Test(expected = MalformedInputException.class)
-	public void testUmlautsInCountryImportNonUTF8() throws IOException, InvalidColumnException, InterruptedException, CsvValidationException, URISyntaxException {
+	@Test
+	public void testUmlautsInCountryImportNonUTF8()
```

Comment:
I don't think this test is needed anymore, since you created tests for other specific encodings. "non UTF8" is a bit vague anyway.

Reply:
Yes. I will clean this up and complete the tests with import files in different encodings and also with larger files to make sure we don't fail for imports that already worked.